Reorg

danaukes opened the pull request:
This is a combination bug-fix & reorganization of the header files to split files along the lines of the objects they demarcate.  
- There are new standard include files and custom include files for putting custom code on forked site instances.  That way, updates to templates can be done in the standard files and updates to site-specific code can be kept separate
- I have created a new, simple way of adding menu items from config.
- The config file is also now used to fill in many pieces of information(organization name, address, email, phone,etc) that had to be hard-coded in the past.
- several navigation bugs have been fixed, specifically the use of "./" links, which fail when you are lower in the directory structure(as in posts)

Please review and let's discuss!

Commit:
moved css and js into assets

Commit:
fixed a typo

Commit:
the local versions are not linked to.

Commit:
fixed bug from moving images

Commit:
fixed js path

Commit:
made new navigation method.

Commit:
moved head content into head.html

Commit:
moved body to default, removed </html> from footer

Commit:
added test navigation pages

Commit:
updated site info automation

Commit:
split up header and footer, made default a little bit more readable.

Commit:
fixed post

Commit:
fixed navbar bug

Commit:
added test post, default layout for post type

Commit:
updated head, added custom head to default

Commit:
added custom foot.

Commit:
changed to generic content.

Commit:
added new layout type.

danaukes committed:
Merge branch 'idealab2' into reorg

merged generic template stuff

danaukes committed:
updated contact info

Conflicts:
	_config.yml
	_includes/footer_custom.html

rraub commented on _includes/footer_custom.html:
```diff
@@ -0,0 +1,13 @@
+  <p>
+    <a class="contact-link" href="mailto:{{site.email}}" id="contact-us-link-in-footer">Contact Us</a>
+  </p>  
+  <a class="btn btn-primary" href="https://www.asufoundation.org/SchoolOfSustainability" id="contribute-button-in-footer">Contribute</a>            
```
We should update this link to just https://www.asufoundation.org/ or directly to https://giving.asu.edu

rraub commented:
Thanks for working on this! We haven't spent a ton of time adding in features to this theme so its good to see it get built out more.

rraub commented on _config.yml:
```diff
@@ -1,13 +1,38 @@
+address: 
+  - Address Line 1
+  - Address Line 2
+  - City, State, AZ 12345
```
The address in the footer is actually getting removed from the standards, it hasn't been formally announced yet but its forthcoming. The reason for this is it has confused Google in the past as to where 'ASU' is. eg: [the honors college](https://barretthonors.asu.edu/) 

Personally I'm a little conflicted with this since our address in the footer gets quite a bit of use internally.

danaukes replied:
It's a little confusing going between the branding site and the stuff your
team puts out, because one appears to be lagging the other.  Is there an
up-to-date reference to show me where the standard is moving to?

Happy to pull that code out if you think it makes sense.

Thanks,

On Tue, Mar 15, 2016 at 9:53 AM, Ryan Raub notifications@github.com wrote:

> In _config.yml
> https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6#discussion_r56198578
> :
> 
> > @@ -1,13 +1,38 @@
> > +address:
> > -  - Address Line 1
> > -  - Address Line 2
> > -  - City, State, AZ 12345
> 
> The address in the footer is actually getting removed from the standards,
> it hasn't been formally announced yet but its forthcoming. The reason for
> this is it has confused Google in the past as to where 'ASU' is. eg: the
> honors college https://barretthonors.asu.edu/
> 
> Personally I'm a little conflicted with this since our address in the
> footer gets quite a bit of use internally.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> 
> https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6/files#r56198578

Commit:
added site.base_path to fix lower-level deployment issues

Commit:
oops, didn't mean to leave that.

Commit:
fixed little bug

rraub commented on _includes/footer.html:
```diff
@@ -14,7 +14,7 @@
           <div class="hidden-sm hidden-xs col-md-2 innovation-footer-image-wrapper">
              <a href="http://yourfuture.asu.edu/rankings" target="_blank" id="best-colleges-us-news-bage-icon">
-              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme{{site.base_path}}/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
+              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
```
We should have this footer image come from this theme rather than the sustainability site.
`{{site.base_path}}/assets/bootstrap-asu/build/img/footer/best-colleges-us-news-badge.png`

danaukes replied:
I think I've implemented all these changes.

On Wed, Mar 16, 2016 at 9:28 AM, Ryan Raub notifications@github.com wrote:

> In _includes/footer.html
> https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6#discussion_r56366067
> :
> 
> > @@ -14,7 +14,7 @@
> >            </div>
> >            <div class="hidden-sm hidden-xs col-md-2 innovation-footer-image-wrapper">
> >               <a href="http://yourfuture.asu.edu/rankings" target="_blank" id="best-colleges-us-news-bage-icon">
> > -              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme{{site.base_path}}/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
> > -              <img src="https://sustainability.asu.edu/wp-content/themes/ASU-Web-Standards-Wordpress-Theme/assets/asu-web-standards/img/footer/best-colleges-us-news-badge.png" alt="Best Colleges U.S. News Most Innovative 2016">
> 
> We should have this footer image come from this theme rather than the
> sustainability site.
> 
> {{site.base_path}}/assets/bootstrap-asu/build/img/footer/best-colleges-us-news-badge.png
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6/files/47d52f043647bf16b9cdaee60350faf558d55aa2..871a66dda013a725a98e2f7cc75fabc1b75eaaf7#r56366067

danaukes committed:
generic template cleanup

navigation bugfix.

danaukes committed:
split universal_nav to its own file for easy modification

updated usnews image to fit rraub's suggestion.
modified donate link
removed usnews.png

danaukes committed:
fixed more nav link base_path problems.

rraub commented:
Looks good! thanks for contributing all these improvements!

danaukes commented:
thanks!

On Wed, Mar 16, 2016 at 12:05 PM, Ryan Raub <notifications@github.com>
wrote:

> Merged #6
> <https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gios-asu/ASU-Web-Standards-Jekyll-Theme/pull/6#event-592343050>
>



-- 
Dan Aukes
idealab.asu.edu

Assistant Professor
The Polytechnic School
Fulton Schools of Engineering
Arizona State University
